### DISCUSSION TITLE
Star count disappearing in new UI

### DISCUSSION THREAD

**Discussion by Sauceke** [community/community#8628] [Other Feature Feedback, Questions, & Ideas] [answered]
Steps to reproduce:
1. Log in
1. Open any repository with an Issues tab
1. Click the Issues tab
1. Click the Code tab

Expected: The number of stars should be visible on the Star button.
Actual: The number of stars disappears from the Star button and does not appear again until the page is reloaded.
Tested with Firefox 95.0 and Edge 96.0.1054.43

**Comment by brianlovin** [comment, marked as answer]
I can repro, thank you for the heads up — and great eye! I'll file an issue for our team 😊

**Reply by Sauceke** [reply]
Thanks, this seems to be fixed now.

**Comment by fredvs** [comment]
Hello.

Sorry but here on Firefox, Click on the number of stars does not show the list of star-users.
In previous release, while Click on the number of stars, a list with each star-user appeared.
Now, nothing appears but it switch from stared<>un-stared.

Or I miss something.

Thanks.
![star_github_nolist](https://user-images.githubusercontent.com/3421249/146823591-e06e667a-1e57-4447-a89b-5cce36511183.png)

**Reply by Wart012** [reply]
> สวัสดี.
> 
> ขออภัย แต่ที่นี่ใน Firefox คลิกที่จำนวนดาวไม่แสดงรายการผู้ใช้ที่ติดดาว ในรุ่นก่อนหน้า ในขณะที่คลิกที่จำนวนดาว รายชื่อผู้ใช้ดาวแต่ละคนก็ปรากฏขึ้น ตอนนี้ ไม่มีอะไรปรากฏแต่มันเปลี่ยนจากการติดดาว<>ไม่ได้ติดดาว
> 
> หรือฉันพลาดอะไรบางอย่าง
> 
> ขอบคุณ. ![star_github_nolist](https://user-images.githubusercontent.com/3421249/146823591-e06e667a-1e57-4447-a89b-5cce36511183.png)

**Reply by brianlovin** [reply]
To view a list of everyone who has starred a repository, you can click this count in the sidebar: 

![Screen Shot 2022-01-04 at 13 06 59@2x](https://user-images.githubusercontent.com/1923260/148124051-398f87c1-a203-44d8-a9e4-42f2c7ca2d59.png)

**Reply by fredvs** [reply]
Hello.

Thanks for the info (that I knew).

So the old behaviour was removed voluntary.

Ok, why not.

**Comment by F-D-X** [comment]
Prosgu

**Comment by netiza** [comment]
Oii
